icu4c/source/test/intltest/measfmttest.cpp

```diff
@@ -4472,12 +4472,7 @@ void MeasureFormatTest::TestIdentifiers() {
         {"exbibyte", "exbibyte"},
         {"zebibyte", "zebibyte"},
         {"yobibyte", "yobibyte"},
-
-        // Testing sort order of prefixes.
-        //
-        // TODO(icu-units#70): revisit when fixing normalization. For now we're
-        // just checking some consistency between C&J.
-        {"megafoot-mebifoot-kibifoot-kilofoot", "kibifoot-mebifoot-kilofoot-megafoot"},
+        {"megafoot-mebifoot-kibifoot-kilofoot", "kilofoot-kibifoot-megafoot-mebifoot"},
```

Comment:
This is not spec compliant - "sort the larger prefixes first" - flip the comparator? (https://www.unicode.org/reports/tr35/tr35-info.html#Unit_Identifier_Normalization)

Reply:
This looks done. Thanks!